projects/travel-tracker.md

```diff
@@ -149,76 +160,118 @@ password: travel
+While M2 rubrics do not have a separate section for WOWs like in M1, there are a few WOW examples noted throughout. In addition to these WOW bullets, you can strive for a WOW by demonstrating not just competency, but excellence and thoroughness across the rubric sections.
+</section>
 
+<section class="answer">
+### Functional Expectations
 
-## Testing
+- Wow: Application fulfills all requirements as well as Iteration 5 or an extension.
+- Yes: Application fulfills all requirements of iterations 1-4 without bugs. **Note: Must be completed in order to pass**
+- Not Yet: Application crashes or has missing functionality or bugs.
+</section>
+
+<section class="answer">
+### UI/UX & Accessibility
+
+* The application can stand on its own to be used by an instructor without guidance from the developer
+* UI/UX is intuitive and easy to read/use
+* Helpful messaging is displayed to prevent user confusion. For example: If a user searches for a room, but none are available then a message is displayed to indicate that the search worked, nothing is broken, there just aren't any matching rooms available.
+* The Lighthouse audit tool was used to improve accessibility.
+* Users who only use keyboards can still navigate this application
+* Users who use a screen reader can still navigate this application
+* Utilizes ARIA attributes on interactive elements when no other tool allows for accessible content
+* Wow option: design is responsive across small, medium and large breakpoints
+</section>
 
-* 4: Application covers all aspects of the application including various flows and covers both happy/sad paths.  Tests must be passing to be considered.
-* 3: Application is well tested but fails to cover some features and only tests for happy paths.  Tests must be passing to be considered.
-* 2: Project has sporadic use of tests at multiple levels. The application contains numerous holes in testing and/or many features are untested.  Tests must be passing to be considered.
-* 1: There is little or no evidence of testing in the application.
+<section class="answer">
+### Fundamental JavaScript and Style / Fetch Competency Examples
+
+* Code is divided into logical components each with a clean, single responsibility
+* Array prototype methods are used to iterate instead of for loops
+* Project is organized to separate concerns (Ask yourself - where should my DOM manipulation live?)
+* Variables and functions are consistently and appropriately named
+* Code leverages JavaScript's truthy/falsey principles
+* Demonstrates efforts towards making functions pure when possible. Note: Purity is not possible for every function in a FE application. Strive for it only when it makes sense.
+* Implements GET and POST Fetch requests
+* When utilizing Fetch, the DOM is updated based on the results of that Fetch
+* Errors are handled and messages are displayed to a user when an error occurs
+* Wow option: Effectively implements one or more closure throughout project.
```

Comment:
Love the updated rubric!  I'm curious, are closures introduced at all in M2?  Apologies if this is an old question!

Reply:
Great question! I had the same one. It looks like the mod will have [this lesson](https://github.com/orgs/turingschool/projects/6/views/1?pane=issue&itemId=20486856) (which hasn't been created yet). 

The M2 team mentioned that they wouldn't expect all students to be able to implement closure yet, but that they could see it being a WOW item.